src/sage/schemes/plane_conics/con_field.py

```diff
@@ -863,40 +865,42 @@ def parametrization(self, point=None, morphism=True):
                     defined by x^2 + x*y + y^2 + x*z + y*z
               To:   Projective Space of dimension 1 over Finite Field of size 2
               Defn: Defined on coordinates by sending (x : y : z) to ...)
-            sage: set(f(p) for p in f.domain())                                         # optional - sage.rings.finite_rings
+            sage: set(f(p) for p in f.domain())
             {(0 : 0 : 1), (0 : 1 : 1), (1 : 0 : 1)}
 
         Verfication of the example ::
 
-            sage: h = g*f; h                                                            # optional - sage.rings.finite_rings
+            sage: # needs sage.rings.finite_rings
+            sage: h = g*f; h
             Scheme endomorphism of Projective Space of dimension 1
              over Finite Field of size 2
               Defn: Defined on coordinates by sending (x : y) to ...
-            sage: h[0]/h[1]                                                             # optional - sage.rings.finite_rings
+            sage: h[0]/h[1]
             x/y
-            sage: h.is_one()  # known bug  (see :trac:`31892`)                          # optional - sage.rings.finite_rings
+            sage: h.is_one()                    # known bug
             True
```

Comment:
Perhaps we keep the related issue number.

Reply:
done in 9d5dfa70bc8